src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -134,10 +134,12 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ proj, enter
     </div>
   );
 
+  const teamName = proj.team ? proj.team.teamName : 'No team assigned';
```

Comment:
ticket says it should say "none"

Reply:
I like no team assigned better actually, none might be a little confusing cause there's nothing else there indicating this is supposed to be a team name. Can you make all the words capital though.